clients/intro_course_developer_component/src/introCourse/pages/SeatAssignment/components/SeatUploader/SeatPlanDialog.tsx

```diff
@@ -27,14 +26,15 @@ export const SeatPlanDialog = ({ seatPlan }: SeatPlanDialogProps): JSX.Element =
           View Seat Plan
         </Button>
       </DialogTrigger>
-      <DialogContent className='max-w-2xl max-h-[80vh] overflow-y-auto'>
-        <DialogHeader>
+      <DialogContent className='max-w-2xl max-h-[90vh] flex flex-col'>
+        {/* Sticky header */}
```

Comment:
I think we can remove the comment here

```suggestion
    <DialogContent className='max-w-2xl max-h-[90vh] flex flex-col'>
```